compiler/control/OMROptions.cpp

```diff
@@ -3833,8 +3869,10 @@ OMR::Options::printOptions(char *options, char *envOptions)
    if (this == TR::Options::getAOTCmdLineOptions())
       optionsType = "AOT";
    TR_Debug::dumpOptions(optionsType, options, envOptions, self(), _jitOptions, TR::Options::_feOptions, _feBase, _fe);
-   if (_aggressivenessLevel > 0)
+   TR_VerboseLog::vlogAcquire();
```

Comment:
Here, write to verbose log is only needed if the if condition evaluates to true. However, we are acquiring and releasing the lock here with a possibility of not using it at all.

Reply:
writelinelocked